homeassistant/components/cover/isy994.py

```diff
@@ -0,0 +1,115 @@
+"""
+Support for ISY994 covers.
+
+For more details about this platform, please refer to the documentation at
+https://home-assistant.io/components/cover.isy994/
+"""
+import logging
+from typing import Callable  # noqa
+
+from homeassistant.components.cover import CoverDevice, DOMAIN
+from homeassistant.components.isy994 import (ISYDevice, NODES, PROGRAMS, ISY,
```

Comment:
So tell me how to fix it - because it works just fine, like I said and I am not a python dev